zimlii/templates/peachjam/_header.html

```diff
@@ -13,6 +13,36 @@
       {% trans 'Legislation' %}
     </a>
   </li>
+  <li class="nav-item dropdown main-nav">
+    <a class="nav-link {% if view.navbar_link == 'taxonomy' %}active{% endif %} dropdown-toggle"
+       href="#"
+       role="button"
+       data-bs-toggle="dropdown"
+       aria-expanded="false">
+      {% trans 'Collections' %}
+    </a>
+    <ul class="dropdown-menu">
+      <li class="dropdown-item">
+        <a class=" nav-link px-0 {% if view.navbar_link == 'legal_instruments' %} active{% endif %} "
```

Comment:
the checks against `navbar_link` can fall away, because they'll never be true. We don't have enough information to know if were on the "Academic papers" taxonomy page.

Ditto for those below.